tests/Aspire.Hosting.Milvus.Tests/MilvusPublicApiTests.cs

```diff
@@ -35,20 +35,6 @@ public void AddMilvusContainerShouldThrowWhenNameIsNull()
         Assert.Equal(nameof(name), exception.ParamName);
     }
 
-    [Fact]
-    public void AddMilvusContainerShouldThrowWhenApiKeyIsNull()
```

Comment:
Can you update 

https://github.com/dotnet/aspire/blob/a7f23ec23cb810adff8c44969be7fcb051b6b871/tests/Aspire.Hosting.Milvus.Tests/MilvusFunctionalTests.cs#L35-L37

To no longer pass in an "apikey" parameter? As that is the default case now.

Then we should have coverage for both passing a parameter (all the existing tests) and for not passing a parameter (in that functional test).